### PR TITLE
Readme example 1 corrections

### PR DESCRIPTION
**Pull request by franz-ka**
Hi there. I've tested and read your code and i liked it. Its simple and easy to read. The only problem is that example 1 in your README has many mistakes. I've corrected them so people can test it more easily :)

If you want i can also add a second node to this example with just a few linesof code to show a complete showcase. As you wish.

Thx 4 sharing <3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,8 +63,8 @@ time.sleep(1)
 node.connect_with_node('127.0.0.1', 8002)
 time.sleep(2)
 
-# Send some message to the other nodes, json style is required!
-node.send_to_nodes({"message": "hoi from node 1"})
+# Send some message to the other nodes
+node.send_to_nodes("hoi from node 1")
```
Comment (macsnoeren) [inline README.md:67]: This send message should work with String, JSON and Binary. It should not be a problem.

**Comment by franz-ka** [comment]
Hi! If I revert the changes and activate debug it shows the following message:
`DEBUG PRINT: Node send_to_node: Error while sending data to the node (debug_print() takes 2 positional arguments but 3 were given)`

It seems as the json feature was something that came obsolete according to a [comment in the code](https://github.com/macsnoeren/python-p2p-network/blob/d31e0bafbdb0c9e4f4ad6d935dc14209807751ee/p2pnetwork/node.py#L140).

Reading the code, i see that it doesn't do any coding/encoding to json with the messages beign sent/received so i think it could be misleading. It also says in a comment in code that data being sent should be [of the type string](https://github.com/macsnoeren/python-p2p-network/blob/d31e0bafbdb0c9e4f4ad6d935dc14209807751ee/p2pnetwork/nodeconnection.py#L54).

**Comment by franz-ka** [comment]
Digging a little bit more i see that `SecureNode` class does implements [json messaging](https://github.com/macsnoeren/python-p2p-network/blob/d31e0bafbdb0c9e4f4ad6d935dc14209807751ee/p2pnetwork/securenode.py#L59), but the example uses `Node` class.

**Comment by franz-ka** [comment]
hey, i revert the json as you asked, but i putted between quotes if not the example crashes. is it right like that? thx

**Comment by macsnoeren** [comment]
Hi Franz-ka,

Could you try to run the unit test to check all the functionality? This is done by python setup.py test. Make sure you use python version 3.

Best regards,

Maurice

**Comment by macsnoeren** [comment]
Hi Frankz-ka, and ... thanks for your interest in the project :).

**Pull request merged**